src/common/sanity.ts

```diff
@@ -58,7 +58,7 @@ export const fetchContent = async <T>(
 };
 
 const fetchContentInternal = async (query: string): Promise<any> => {
-  const response = await fetch(queryUrl(query));
+  const response = await fetch(queryUrl(query), { credentials: "include" });
```

Comment:
Oops, unrelated change.

```suggestion
  const response = await fetch(queryUrl(query));
```